README.md

```diff
@@ -372,6 +390,9 @@ _X needs to be replaced with 0 or 1 depending upon firmware0 or firmware1 respec
 
 ##System upgrade
 
+If booted using any of the above methods, then you can upgrade your OpenWrt image using sysupgrade utility.
```

Comment:
You can get rid of "If booted using any of the above methods, then" part, the next sentence "However please note" covers what you want to convey